blocks/identity-block/components/login/index.jsx

```diff
@@ -57,13 +67,15 @@ const useLogin = ({
 			const redirectUrlLocation = new URL(document.referrer);
 
 			if (searchParams.has('reset_password')) {
-				setCurrentRedirectToURL(`${redirectURL}${redirectUrlLocation.search}`);
+				const newRedirectUrl = `${redirectURL}${redirectUrlLocation.search}`;
+
+				setRedirectUrl(newRedirectUrl);
```

Comment:
@accbjt  I think we can define a `let newRedirectUrl` and move this line out from the if-else statement.

Reply:
@LauraPinilla Okay, I made those updates.